src/e2e/app/cli/repo-tests/src/test_auth.rs

```diff
@@ -166,3 +166,125 @@ pub async fn test_login_dummy_github(kamu_api_server_client: KamuApiServerClient
 }
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+pub async fn test_kamu_access_token_middleware(kamu_api_server_client: KamuApiServerClient) {
+    // 1. Grub a JWT
+    let login_response = kamu_api_server_client
+     .graphql_api_call(indoc::indoc!(
+        r#"
+            mutation {
+                auth {
+                    login(loginMethod: "password", loginCredentialsJson: "{\"login\":\"kamu\",\"password\":\"kamu\"}") {
+                        accessToken,
+                        account {
+                            id
+                        }
+                    }
+                }
+            }
+        "#,
```

Comment:
Minor: formatting 

Since we're using `indoc! `for the sake of cutting out extra leading spaces, we shouldn't add them ourselves then :)

I mean:
```
        r#"
        mutation {
            auth {
                login(loginMethod: "password", loginCredentialsJson: "{\"login\":\"kamu\",\"password\":\"kamu\"}") {
                    accessToken,
                    account {
                        id
                    }
                }
            }
        }
        "#
```

Reply:
![image](https://github.com/kamu-data/kamu-cli/assets/29308899/ae88e7f1-fa95-4d73-8fc7-bcf1a7947c95)